lib/plugins/aws/package/compile/events/apiGateway/index.js

```diff
@@ -262,6 +268,18 @@ class AwsCompileApigEvents {
               'to "provider.apiGateway"'
           );
         }
+
+        if (
+          this.serverless.service.provider.name === 'aws' &&
+          Object.values(this.serverless.service.functions).some(({ events }) =>
+            events.some(({ http }) => _.get(http, 'request.schema'))
+          )
+        ) {
+          this.serverless._logDeprecation(
+            'AWS_API_GATEWAY_SCHEMAS',
+            'Starting with next major version, `http.request.schema` property will be replaced by `http.request.schemas`.'
```

Comment:
In CLI logs, we usually do not use back quotes, but double quotes. They look better for wrapping configuration properties

Reply:
Good catch - fixed :+1: